### ISSUE TITLE
[Suggestion] Duplicate but separate configs per dimension

### ISSUE THREAD

**Issue by Darkmega18** [mactso/SpawnBalanceUtility#21]
I would be interesting if you'd create a different config for each dimension detected in the modpack and sort them by folders and allow each dimension to have it's own set of configs.

For example, a mod might add the hex world or the forgotten cities but in a separate dimension rather than overriding the overworld.
with this idea, you'd be able to have a spawn rate for one of those worlds. but then in another world you could configure it so that normal mobs wouldn't spawn but more dangerous mobs from another mod for example would spawn a lot more frequently making that dimension more deadly.

Structure spawns and mob entitities would probably still stay in the root of the config folder.

**Comment by mactso** [comment]
This would normally be handled by Biome.   Each dimension has it's own set of custom Biomes.

Are you saying there are dimensions which use the same biomes ( like "minecraft:plains"?)

Thanks,
MacTso

**Comment by Darkmega18** [comment]
Yeah, there are occasionally mods which for example just generate a different kind of overworld.
For example theres normal overworld, but then you can get to the forgotten cities dimension, which is overworld like but it has cities overlayed on it.
Then there might be something like hexworld, where the biomes are giant hexagonal tiles etc. or a mod that generates an exact copy of the overworld, like Second world.
finally there are sometimes the really fancy mods like mystcraft or RFtools dimensions that can generate custom worlds with existing biomes.

So it's situational, but there do exist some such mods.

**Comment by mactso** [comment]
I love hexworld.  :)

I'll give this some thought.   It does seem pretty hard but maybe I'll think of something.
